app/javascript/components/common/index.ts

```diff
@@ -1 +1,5 @@
 export { CopyToClipboardButton } from './CopyToClipboardButton'
+export { GraphicalIcon } from './GraphicalIcon'
+export { Icon } from './Icon'
+export { Loading } from './Loading'
```

Comment:
What does this LOC do?

Reply:
It is a React convention for the `index.ts` to export all the components in the directory. You can then do the following to emulate some form of namespacing:

```jsx
import * as Common from '../components/common'

<Common.CopyToClipboardButton textToCopy={data.text_to_copy} />
```

Reply:
Ah, cool.